src/app/wesprzyj/platnosc/_components/StripeElementsWrapper.tsx

```diff
@@ -0,0 +1,78 @@
+"use client";
+
+import { Elements } from "@stripe/react-stripe-js";
+import { loadStripe, type StripeElementsOptions } from "@stripe/stripe-js";
+
+const stripePublishableKey = process.env.NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY;
+
+if (!stripePublishableKey) {
+  throw new Error("Missing NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY env variable");
+}
+
+const stripePromise = loadStripe(stripePublishableKey);
+
+interface Props {
+  clientSecret: string;
+  children: React.ReactNode;
+}
+
+export const StripeElementsWrapper = ({ clientSecret, children }: Props) => {
+  const options = {
+    clientSecret,
+    appearance: {
```

Comment:
Let's move the static values outside the component and join it with the `clientSecret` here